src/core/connection.c

```diff
@@ -2520,13 +2522,13 @@ QuicConnSetConfiguration(
         if (!QuicConnPostAcceptValidatePeerTransportParameters(Connection)) {
             QuicConnTransportError(Connection, QUIC_ERROR_CONNECTION_REFUSED);
             Status = QUIC_STATUS_INVALID_PARAMETER;
-            goto Error;
+            goto Cleanup;
```

Comment:
Why do we have both `Cleanup` and `Error` labels if we sometimes go to `Cleanup` without generating `LocalTP` parameters?